federation/apis/federation/register.go

```diff
@@ -48,7 +48,6 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Cluster{},
 		&ClusterList{},
 		&api.ListOptions{},
-		&api.DeleteOptions{},
```

Comment:
I see. Anyway, there should be no need to modify the API in this PR. We are not deleting anything here.